cohorts/load.py

```diff
@@ -840,6 +844,13 @@ def get_filtered_isovar_epitopes(self, epitopes, ic50_cutoff):
                 mutant_binding_predictions.append(binding_prediction)
         return EpitopeCollection(mutant_binding_predictions)
 
+    def load_ensembl_coverage(self, min_depth=30):
+        if self.pageant_coverage_path is None:
```

Comment:
comment on what this path should look like? It should be a folder of patient id's it seems? with a `cdf.csv` file in that dir?